drivers/sensor/tmp112/Kconfig

```diff
@@ -12,3 +12,10 @@ config TMP112
 
 	  The TMP102 is compatible with the TMP112 but is less accurate and has
 	  been successfully tested with this driver.
+
+config TMP112_FULL_SCALE_RUNTIME
+	bool "Allow conversion rate at runtime"
+
+config TMP112_SAMPLING_FREQUENCY_RUNTIME
+	bool "Allow to set extended mode at runtime"
+
```

Comment:
Putting this part here means that once selected all instance will be configurable at runtime. Is this what you want? I guess it is done for simplicity, and I'm not against it.

Reply:
Yes that's true. This is done to safe some resources (flash) when the sensor configuration does not need to be changed after initialization which I think is the most common use-case. That said for backward compatibility it would be better to have it activated by default, like the older version. This has been lost in the rebase